ccd-config-generator/src/main/java/uk/gov/hmcts/ccd/sdk/type/Organisation.java

```diff
@@ -0,0 +1,14 @@
+package uk.gov.hmcts.ccd.sdk.type;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Data;
+import uk.gov.hmcts.ccd.sdk.api.ComplexType;
+
+@Data
+@Builder
+@ComplexType(name = "Organisation", generate = false)
+public class Organisation {
```

Comment:
Should we add @AllArgsConstructor and @NoArgsConstructor.  If this is used for Json generation this will fail due to the @Builder annotation setting the no args constructor to private

Reply:
A no args constructor won't work because of the final value but I'll add the all args